src/sentry/static/sentry/app/views/organizationIntegrations/integrationRepos.jsx

```diff
@@ -149,7 +149,7 @@ export default class IntegrationRepos extends AsyncComponent {
     let items = (repositories || []).map(repo => {
       return {
         searchKey: `${repo.name}`,
-        value: `${repo.full_name}`,
+        value: `${repo.identifier}`,
```

Comment:
Can you pull these out of the backticks while your in here.. there's no need for the string interpolation